components/Tabs/Tabs.tsx

```diff
@@ -105,48 +119,54 @@ export const Tabs = ({
    */
   const uniquePrefix = `${id || tabGroupId}-`;
 
+  const tabItemsWithSlug = tabItems.map((tabItem) => ({
+    ...tabItem,
+    slug: slugify(tabItem.label),
+  }));
```

Comment:
Instead of calling slugify in so many places, I just add the slug once and for all to the tabItems array